### DISCUSSION TITLE
Open Completions Panel Not Showing

### DISCUSSION THREAD

**Discussion by Jevxtn** [community/community#67473] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I am following the Github Copilot module part Code with Github Copilot inside Codespace(web). On step 3 When I was trying to do that step I couldn't do so since the option is not showing up in the right click menu from the suggestion.

Thanks

![copilot](https://github.com/community/community/assets/144324850/0af70cbf-ed71-4fc3-9e4b-317e8a559a32)

**Comment by Durgesh4993** [comment]
Just in case anyone else has the same problem here is how I resolved my issue. I won't bore you with how I discovered the solution.
Right-click in the activity bar, this will bring up a context menu with the entry for Open Editors listed but not marked with a tick
[Activity bar context menu](https://i.stack.imgur.com/7RkuP.png).

Clicking on "Open Editors" in this context menu brings up a new icon in the activity bar that looks like an open book
[Open Editor icon](https://i.stack.imgur.com/G6VnV.png)

Right-clicking on this new icon reveals another context menu that includes the option to reset the location of the Open Editors panel.
[Open Editor context menu](https://i.stack.imgur.com/Jp3eH.png)

Selecting this option resolves the problem.

How I got into the problem in the first place is still difficult to ascertain. With the activity bar in its default configuration I must have dragged the "Open Editors" header to somewhere on the activity bar, that clearly removes it from the Explorer side bar but how I inadvertently right clicked this new icon and selected "Hide 'Open Editors'" without realising remains a mystery.

**Reply by GaganaMD** [reply]
I just tried this method. I am still not able to locate "Open completions panel".

**Reply by Jevxtn** [reply]
Same here

**Reply by GaganaMD** [reply, marked as answer]
Hey! I was able to bring up the GitHub Copilot completions panel by using: control + enter

Hope this works for you as well.

**Reply by Jevxtn** [reply]
It works! Big help @GaganaMD Thanks.

**Comment by sureshreddysiddana** [comment]
Hi, control + enter is working to bring up **GitHub Copilot Suggestions** Window
![image](https://github.com/community/community/assets/105904978/a4dc160e-4268-4e3c-8d27-1045a5b729b0)
